modules/dxp/apps/osb/osb-faro/osb-faro-docker/local/Dockerfile

```diff
@@ -25,7 +25,6 @@ HEALTHCHECK --interval=3s --timeout=3s CMD curl -f http://localhost:8080 || exit
 RUN mkdir -p $LIFERAY_HOME/tomcat/lib/org/apache/catalina/util && \
     echo "server.info=" > $LIFERAY_HOME/tomcat/lib/org/apache/catalina/util/ServerInfo.properties
 
-COPY --chown=liferay:liferay common/license $LIFERAY_HOME/deploy
```

Comment:
Hey @ivicac, I think we still need a license file when starting up AC locally (we do for uat and prd environments), so we will need to add the license file manually into the common directory. Because of this, I think we still need this line